build/docker/data/assets/script/load_dos_db.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh.
```

Comment:
Can we place that file in the `build/docker/data/assets/sbin/init.d/` directory and rename it to `00-load-database.sh`?